### ISSUE TITLE
Unblocked domains are not reachable

### ISSUE THREAD

**Issue by uservictor** [AdguardTeam/AdGuardHome#3252]
Hello I use v0.107.0-a.65+4700dea9 on RaspberryPi.

Problem description:
If I want to reach an blocked (due filter list) domain, I usually go to Query Log and unblock the neccessary domain. The problem is that the domain is still not reachable due the DNS cache.

Solution:
If I disable DNS Cache (delete the cache size value) everything works, I can see the website right away.

Maybe its possible after unblock an domain (@@||examle.org^$important) to check the cached entrys for matches and removing them.

Thanks

**Comment by Aikatsui** [comment]
Duplicate of #1145
